packages/block-library/src/social-link/edit.js

```diff
@@ -40,6 +69,7 @@ const SocialLinkURLPopover = ( {
 				event.preventDefault();
 				setPopover( false );
 			} }
+			ref={ useOnDelete( { url, clientId } ) }
```

Comment:
Why an event handler on the form and not the input?

Reply:
It would be the more correct solution though. Maybe other blocks want to do something similar? Why can't they pass the same props as they would on a normal input field? Can you pass a ref instead?

Reply:
Ok, I'll add the usage of `onKeyDown` in `URLInput`.